velox/dwio/dwrf/test/WriterExtendedTests.cpp

```diff
@@ -474,5 +474,347 @@ TEST_F(E2EWriterTest, FlushPolicyFlatMap) {
   }
 }
 
+TEST_F(E2EWriterTest, MemoryPoolBasedFlushPolicySimpleEncoding) {
```

Comment:
```
s/MemoryPoolBasedFlushPolicySimpleEncoding/memoryPoolBasedFlushPolicySimpleEncoding/
```

Reply:
We have used this convention for this half of the code base though.